### ISSUE TITLE
Bug: Node selection menu disappears if 25/100 row per page is selected for a node with many channels

### ISSUE THREAD

**Issue by tukac** [Ride-The-Lightning/RTL#619]
**Describe the bug**
I run multiple nodes. Top left drop-down menu for node selection disappears if 25 or 100 channel view is selected for a node with many channels. However, if there is sufficient space on a page to show all channels, than selecting 25/100 view doesn't disappear drop-down menu. Therefore this bug is only if many channels are open, key issue probably being vertical space available in a browser.

**To Reproduce**
Run multiple nodes, have more open channels than there is vertical space on the page, select 25 or 100 view option.

**Your environment**
* Version of `RTL` 0.10.0
* Browser & browser version  Firefox 86.0
* Operating system (`uname -a` on *Nix) Linux Mint

**Additional context**
Returning to 10 channel view does not returns the channel selection menu, neither does refreshing. Firefox needs to restart for menu to re-appear!

**Additional feature request:** 
-
Selection of 25/100 channel view is not memorized! 10 channel view always returns back, even within same session. Please make view selections memorized, preferably even after browser restart.

**Comment by tukac** [comment]
Additional bug with top-left dropdown menu for node selection: after node renaming the menu seems to keep original node name and does not update new name. I have done it only once but node name is still old after many system restarts. New name has  utf-8 character, old had alphanumerics only.

**Comment by ShahanaFarooqui** [comment]
Node selection dropdown disappear bug: It has been fixed with PR #645 (by reducing mat-sidenav-container height in root.scss file.)

Node name does not update: I failded to rerpoduce it with below steps:
    1: Update the node name into RTL-Config.json file (including utf-8 character).
    2: Restart RTL server or service (node rtl).
    3: Refresh the browser and login to the UI. 

Closing this issue in account of the first bug. Please feel free to open another issue with reproduction steps if the node name does not update after following above steps.
